Gordon360/Services/ScheduleService.cs

```diff
@@ -24,18 +24,25 @@ public ScheduleService(CCTContext context)
         /// Fetch the schedule item whose id and session code is specified by the parameter
         /// </summary>
         /// <param name="username">The AD Username of the student</param>
+        /// <param name="sessionID">The selected session of the student</param>
         /// <returns>StudentScheduleViewModel if found, null if not found</returns>
-        public async Task<IEnumerable<ScheduleViewModel>> GetScheduleStudentAsync(string username)
+        public async Task<IEnumerable<ScheduleViewModel>> GetScheduleStudentAsync(string username, string? sessionID)
         {
             var account = _context.ACCOUNT.FirstOrDefault(x => x.AD_Username == username);
 
             if (account == null)
             {
-                throw new ResourceNotFoundException() { ExceptionMessage = "The Schedule was not found." };
+                throw new ResourceNotFoundException() { ExceptionMessage = "The Account was not found." };
             }
```

Comment:
I know y'all didn't write this, but sine we're updating these methods, here's an opportunity to improve the code. Instead of selecting directly from the `ACCOUNT` view, you can use the IAccountService, which implements the Exception if not found for us, and decouples the code further. That would look like this (although you may have to add IAccountService as a param of this class if it isn't already):
```csharp
var account = _accountService.GetAccountByUsername(username);
```